### ISSUE TITLE
The settings of dataset

### ISSUE THREAD

**Issue by taibai123abc** [Ghostish/Open3DSOT#47]
In m2track's configuration file, you've set the validation set and test set to be the same, is this a convention? This seems to have a significant impact on the results.

**Comment by Ghostish** [comment]
Yes, it is a common setup. You can also use a smaller subset to do the validation if you want to save training time.
